modules/gapi/include/opencv2/gapi/infer/ie.hpp

```diff
@@ -117,75 +140,180 @@ template<typename Net> class Params {
               , 1u} {
     };
 
-    Params<Net>& cfgInputLayers(const typename PortCfg<Net>::In &ll) {
+    /** @brief Specifies sequence of network input layers names for inference.
+
+    The function is used to associate data of graph inputs with input layers of
+    network topology. Number of names has to match the number of network inputs. If a network
```

Comment:
> Number of names has to match the number of network inputs

Is it? it should match the number of network inputs as defined in `G_API_NET()` but not as in the model itself.

Reply:
Done